### PR TITLE
SlateDB bootstrapping agreement, 2024-11-05

### PR DESCRIPTION
**Pull request by criccomini**

### CODE REVIEW

**Review by gavinking — COMMENTED** [review]
Should SlateDB also be added to the list of unregistered trademarks in `TRADEMARKS.md`?

File: TRADEMARKS.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ The Commonhaus Foundation uses the following unregistered trademarks:
     <img src="https://github.com/quarkusio/quarkus/blob/main/core/deployment/src/main/resources/META-INF/branding/logo.png?raw=true" alt="" height="22" />
 - SDKMAN! Logo &trade; [^1]  
     <img src="https://sdkman.io/assets//img/sdk-man-small-pattern.svg" alt="" height="22" />
+- SlateDB Logo &trade;  [^1]
+    <img src="https://github.com/slatedb/slatedb-website/blob/main/assets/svg/icon1.svg" alt="" height="22" />
```
Comment (ebullient) [inline TRADEMARKS.md:68]:
```suggestion
    <img src="https://github.com/slatedb/slatedb-website/blob/main/assets/svg/icon1.svg?raw=true" alt="" height="22" />
```

**Comment by criccomini** [comment]
> Should SlateDB also be added to the list of unregistered trademarks in `TRADEMARKS.md`?

Ah, yes I think so.

**Comment by criccomini** [comment]
Resolved the missing unregistered text trademark here: [bce03ea](https://github.com/commonhaus/foundation/pull/212/commits/bce03eae00c157086293732ee35e7773c877ac4a)

**Comment by gavinking** [comment]
Thanks, LGTM now.

**Review by ebullient — APPROVED** [review]

**Comment by ebullient** [comment]
changes look good. internal discussion in the meanwhile.

**Review by gavinking — APPROVED** [review]

**Pull request merged**